lib/dynamo-backup.js

```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
         if (err) {
             return callback(err);
         }
+        var limit = 1;
 
-        var readPercentage = self.readPercentage;
-        var limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
+        if (data.Table.BillingModeSummary.BillingMode == "PAY_PER_REQUEST") {
+            // default to 200 RCUs if we are in pay per request
+            limit = 200;
+        } else {
+            var readPercentage = self.readPercentage;
+            limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
```

Comment:
lol Math.max(false, 10) // 10
but Math.max(NaN, 10) // NaN 
🤦‍♂ JS sucks sometimes
Good pickup here :)

Reply:
I didn't - thats the original code :)